tdrs-backend/tdpservice/stts/urls.py

```diff
@@ -0,0 +1,10 @@
+"""Routing for STTs."""
+
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("by_region", views.RegionAPIView.as_view(), name="stts-by-region"),
+    path("alpha", views.STTApiAlphaView.as_view(), name="stts-alpha"),
+    path("", views.STTApiView.as_view(), name="stts")
```

Comment:
do we have a sense of the use case for an unordered list endpoint?

Reply:
ATM I am not sure, it just fit with a standard RESTful approach. Anyone down the road consuming a RESTful API would expect a GET at /stts to return a list of all of the STTs in the system. That was the reasoning, and it only took a few minutes to stand up.